ui/raidboss/data/03-hw/alliance/weeping_city.js

```diff
@@ -0,0 +1,312 @@
+      condition: function(data) {
+        return data.arachneStarted;
+      },
+      response: Responses.stackOn(),
+    },
+    {
+      id: 'Weeping City Frond Affeared',
+      netRegex: NetRegexes.startsUsing({ id: '183A', source: 'Arachne Eve', capture: false }),
+      response: Responses.lookAway(),
+    },
+    {
+      id: 'Weeping City Arachne Web',
+      netRegex: NetRegexes.headMarker({ id: '0017' }),
+      condition: function(data, matches) {
+        return data.arachneStarted && data.me == matches.target;
+      },
+      infoText: {
+        en: 'Drop Web Outside',
+      },
+    },
+    {
+      id: 'Weeping City Brand Of The Fallen',
+      netRegex: NetRegexes.headMarker({ id: '0037' }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.doritoStack(),
+    },
+    {
+      id: 'Weeping City Dark Eruption',
+      netRegex: NetRegexes.headMarker({ id: '0019' }),
+      condition: Conditions.targetIsYou(),
+      infoText: {
+        en: 'Puddles on YOU',
+        de: 'Pfützen auf DIR',
+        fr: 'Mare sur VOUS',
+        ja: '自分に床範囲',
+        cn: '点名',
+        ko: '장판 바깥에 깔기',
+      },
+    },
+    {
+      id: 'Weeping City Beguiling Mist',
+      netRegex: NetRegexes.startsUsing({ id: '17CE', source: 'Summoned Succubus' }),
+      condition: function(data) {
+        return data.canSilence();
+      },
+      response: Responses.interrupt(),
+    },
+    {
+      id: 'Weeping City Mortal Ray',
+      netRegex: NetRegexes.startsUsing({ id: '17D4', source: 'Summoned Haagenti' }),
+      response: Responses.lookAwayFrom(),
+    },
+    {
+      id: 'Weeping City Hell Wind',
+      netRegex: NetRegexes.startsUsing({ id: '17CB', source: 'Forgall', capture: false }),
+      // Hell Wind sets HP to single digits, so mitigations don't work. Don't notify non-healers.
+      condition: function(data) {
+        return data.role == 'healer';
+      },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Weeping City Mega Death',
+      netRegex: NetRegexes.startsUsing({ id: '17CA', source: 'Forgall', capture: false }),
+      alertText: {
+        en: 'Get 1 Stack In Puddle',
+      },
+    },
+    {
+      id: 'Weeping City Meteor Impact',
+      netRegex: NetRegexes.headMarker({ id: '0039' }),
+      condition: Conditions.targetIsYou(),
+      alertText: {
+        en: 'Drop meteor back or left',
+      },
+    },
+    {
+      // The ability used here is Ozma entering Pyramid form.
+      // Execration follows this up almost immediately.
+      id: 'Weeping City Execration',
+      netRegex: NetRegexes.ability({ id: '1826', source: 'Ozma', capture: false }),
+      alertText: {
+        en: 'Get off rectangle platform',
+      },
+    },
+    {
+      // The ability used here is Ozma entering Cube form.
+      // Flare Star and Explosion follow shortly.
+      id: 'Weeping City Flare Star',
+      netRegex: NetRegexes.ability({ id: '1803', source: 'Ozma', capture: false }),
```

Comment:
I've seen tanks gets orbs too.  Would it make sense to split this into:
Laser cleaves on you (tanks) / avoid tank lasers (others)
Orbs (tanks/healers) / Avoid Orbs (dps)

Reply:
That's doable. We can notify for tank lasers on Cube form, and then notify to pop orbs when they spawn.